testsuite/test_meeting_processing_file_list.py

```diff
@@ -0,0 +1,127 @@
+                                  meeting_id=meeting.id))
+        assert resp.status_code == 200
+        html = PyQuery(resp.data)
+        assert html('#job-list tbody tr').length == 2
+
+
+def test_superuser(app):
+    meeting = MeetingFactory()
+
+    first_role = RoleFactory(permissions=('',))
+    second_role = RoleFactory(permissions=('view_participant',))
+
+    first_role_user = RoleUserMeetingFactory(role=first_role,
+                                             meeting=meeting)
+    second_role_user = RoleUserMeetingFactory(role=second_role,
+                                              meeting=meeting)
+
+    first_role_user.user.is_superuser = True
+
+    first_job = JobFactory(user_id=first_role_user.user.id,
+                           meeting_id=first_role_user.meeting.id)
+    second_job = JobFactory(user_id=second_role_user.user.id,
+                            meeting_id=first_role_user.meeting.id)
+
+    db.session.add(first_job)
+    db.session.add(second_job)
+    db.session.commit()
+
+    client = app.test_client()
+    with app.test_request_context():
+        with client.session_transaction() as sess:
+            sess['user_id'] = first_role_user.user.id
+
+        resp = client.get(url_for('meetings.processing_file_list',
+                                  meeting_id=meeting.id))
+        assert resp.status_code == 200
+        html = PyQuery(resp.data)
+        assert html('#job-list tbody tr').length == 2
+
+
+def test_view_participant_user(app):
+    meeting = MeetingFactory()
+
+    first_role = RoleFactory(permissions=('view_participant',))
+    second_role = RoleFactory(permissions=('view_participant',))
+
+    first_role_user = RoleUserMeetingFactory(role=first_role,
+                                             meeting=meeting)
+    second_role_user = RoleUserMeetingFactory(role=second_role,
+                                              meeting=meeting)
+
+    first_job = JobFactory(user_id=first_role_user.user.id,
+                           meeting_id=first_role_user.meeting.id)
+    second_job = JobFactory(user_id=second_role_user.user.id,
+                            meeting_id=first_role_user.meeting.id)
+
+    db.session.add(first_job)
+    db.session.add(second_job)
+    db.session.commit()
+
+    client = app.test_client()
+    with app.test_request_context():
+        with client.session_transaction() as sess:
+            sess['user_id'] = first_role_user.user.id
+
+        resp = client.get(url_for('meetings.processing_file_list',
+                                  meeting_id=meeting.id))
+        assert resp.status_code == 200
+        html = PyQuery(resp.data)
+        assert html('#job-list tbody tr').length == 1
+
+
+def test_user_without_permissions(app, user):
+    role = RoleFactory(permissions=('',))
+    role_user = RoleUserMeetingFactory(role=role)
+
+    job = JobFactory(user_id=role_user.user.id,
+                     meeting_id=role_user.meeting.id)
```

Comment:
In order to test this case, you only need the user with no role and a meeting. A user without any permissions assigned won't be able to access this page. If, however, you want to add some jobs, they cannot pertain to this user.